src/pathhelper.cpp

```diff
@@ -12,12 +12,12 @@ namespace PathHelper
 
     QString GetFiltersConfigPath()
     {
-        return GetConfigPath() + QDir::separator() + QStringLiteral("filters");
+        return GetConfigPath() + "/" + QStringLiteral("filters");
```

Comment:
Why? It is supposed to be OS specific.

Reply:
From the QT documentation:

```
Qt uses "/" as a universal directory separator in the same way that "/" is used as a path separator in URLs. If you always use "/" as a directory separator, Qt will translate your paths to conform to the underlying operating system.
```

I don't see a good reason not to stick to the QT convention of using '/' everywhere.
Also,  GetConfigPath() itself already returns "/" instead of "\" on Windows. Hence, we so far had mixed '/' and '\' in our paths.

In case we want the app to feel more native, we should rather convert the paths just before displaying them to the user